docs/assets/js/back-to-top-smp.js

```diff
@@ -0,0 +1,91 @@
+// Crafted with ☕ for Service Mesh Performance
+// © Neilblaze 2020

```

Comment:
Layer5 projects use Apache v2 licensing. We'll need to either remove this line or include this standard license header instead:

```
Copyright 2020 Layer5 Authors

Licensed under the Apache License, Version 2.0 (the "License"); 
you may not use this file except in compliance with the License. 
You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software 
distributed under the License is distributed on an "AS IS" BASIS, 
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
See the License for the specific language governing permissions and 
limitations under the License.
```

Reply:
Oops, sorry, actually I forgot to erase this part :laughing:

Reply:
Removed that part!!! :smile:

Reply:
It will be good to include this licensing in the file. @Neilblaze

Reply:
Included that!